README.md

```diff
@@ -1102,3 +1116,24 @@ Check out the demo in the folder `/manifests/regex-parser` for how to use a cust
 ### Manifests
 
 Most files under the folder [manifests/setup](manifests/setup) are automatically generated from [config](config). Don't edit them directly, run `make manifests` instead, then replace these files accordingly.
+
+## Community
+
+### About the community
+
+Fluent Operator is sponsored and open-sourced by the [KubeSphere](http://kubesphere.io/) Team and maintained by the Fluent community. This project was accepted by the Fluent community as a sub-project in 2021 and is currently a vendor-neutral and community-driven project. 
+
+### Contact us
+
+You are welcome to join us in the Fluent community to polish it together:
+
+- Slack: Create an account in the [Fluent Slack workspace](https://launchpass.com/fluent-all) and then search `#fluent-operator` to join or click [#fluent-operator](https://fluent-all.slack.com/archives/C02642TE9HR) to join if you already have an account
+- Wechat: join the KubeSphere Observability SIG by following the `kubesphere` WeChat subscription
```

Comment:
I was a bit confused initially as there is a Kubernetes sig as well for this and mis read it. Do we have any involvement in that?

Reply:
Not yet. WeChat subscription and group are only used for Chinese users since we also have a Chinese community and SIG Observability to run this project and communication regularly.